src/commands/project/delete/source.ts

```diff
@@ -214,14 +214,14 @@ export class Source extends SfCommand<DeleteSourceJson> {
         this.components
           .filter((comp) => comp.type.strategies?.adapter === 'bundle')
           .filter(isSourceComponent)
-          .map((bundle: SourceComponent) => {
```

Comment:
the curly bracket here causes the stuff inside to execute, but it should have *returned* the result.